Tools/Scripts/delete-if-too-large

```diff
@@ -25,8 +25,18 @@
 
 import sys
 import os
+import subprocess
 from pathlib import Path
 
+def get_folder_size(folder):
```

Comment:
Would it be reasonable to shell out to `du` instead? We're already shelling out to `rm` for deleting anyway, not using native Python code.